package-lock.json

```diff
@@ -1,10 +1,12 @@
 {
   "name": "elastic-otel-node",
+  "version": "0.1.0",
```

Comment:
note for reviewer: at 1st I though this line was added because I was using a different node version but it happened in v14, v16, v18 & v20. Is okay this addition?